src/Bot.hs

```diff
@@ -203,12 +205,20 @@ builtinCommands =
     , ( "wiggle"
       , ( "Wiggle the tenticle (integration with https://github.com/tsoding/wiggle)"
         , transR (Identity . messageSender) $
-          cmapR (URI.encode . T.unpack . senderName) $
-          Reaction $ \(Identity name) -> do
-            request <- parseRequest [qms|http://localhost:8081/wiggle/{name}|]
-            void $ httpRequest request))
+          cmapR (URI.encode . T.unpack . senderName) wiggle))
     ]
 
+wiggle :: Reaction Identity String
```

Comment:
I'd suggest to turn wiggle into `Comonad w => Reaction w String` and use [extract] to unwrap the name. That will make the function reusable accross multiple Comonads.

[extract]: https://hackage.haskell.org/package/comonad-5.0.4/docs/Control-Comonad.html#v:extract